webpack/webpack-plugin-utilities/README.md

```diff
@@ -0,0 +1,49 @@
+# webpack-plugin-utilities
+
+## Installation
+
+`npm install @rushstack/webpack-plugin-utilities --save-dev`
```

Comment:
```suggestion
`npm install @rushstack/webpack-plugin-utilities --save`
```

If someone uses this in a webpack plugin project, they need it to be a regular dependency.